src/MarkdownEditor.js

```diff
@@ -65,7 +78,7 @@ var MarkdownEditor = React.createClass({
     };
 
     if(this.props.styles.styleMarkdownEditorHeader) {
-      styleMarkdownEditorHeader = this.props.styles.styleMarkdownEditorHeader;
+      Object.assign(styleMarkdownEditorHeader, this.props.styles.styleMarkdownEditorHeader);
```

Comment:
Would it be best to use [babel-polyfill](https://babeljs.io/docs/usage/polyfill/), [babel-transform-object-assign](https://babeljs.io/docs/plugins/transform-object-assign/), or just create a 'polyfill' file in utils?